projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed-hlist-item/external-feed-hlist-item.component.scss

```diff
@@ -0,0 +1,95 @@
+ @import '/projects/commudle-admin/src/assets/styles/variables.scss';
+
+ .external-feed-list-item {
+
+   p,
+   a,
+   div,
+   h3 {
+     font-family: $primary-font-family;
+   }
+
+   nb-card {
+     margin-bottom: 0.5rem;
+
+     nb-card-body {
+       &:hover {
+         box-shadow: 1px 1px 5px 5px $lightgrey;
+       }
+
+       .image {
+         border: 1px solid $lightestgrey;
+         background-color: $lightestgrey;
+         width: 100%;
+         height: 140px;
+         overflow: hidden;
+         margin-bottom: 5px;
+
+         img {
+           width: 100%;
+         }
+       }
+
+       .source-details {
+         align-items: center;
+         display: flex;
+
+         .external-feed-post-source-image {
+           min-height: 2.5rem;
+           max-height: 2.5rem;
+           min-width: 2.5rem;
+           max-width: 2.5rem;
```

Comment:
In case of fixed size of height and width, source images are coming of different sizes. In the original one, all the images are uniform. Should this change be made ?